wordpress-mysql-replication/nested/mysql-replication.json

```diff
@@ -225,12 +225,12 @@
     "customScriptFilePath": "[concat(parameters('artifactsPath'), '/scripts/azuremysql.sh')]",
     "mysqlConfigFilePath": "[concat(parameters('artifactsPath'), '/scripts/my.cnf.template')]",
     "singleQuote": "",
-
     "virtualNetworkSetupURL": "[concat(parameters('artifactsPath'),'/nested/vnet-',parameters('vnetNewOrExisting'),'.json')]",
     "sa": "[parameters('dbSubnetStartAddress')]",
     "ipOctet01": "[concat(split(variables('sa'), '.')[0], '.', split(variables('sa'), '.')[1], '.')]",
     "ipOctet2": "[int(split(variables('sa'), '.')[2])]",
-    "ipOctet3": "[int(split(variables('sa'), '.')[3])]"
+    "ipOctet3": "[int(split(variables('sa'), '.')[3])]",
+    "DbName":"[concat(uniquestring(resourceGroup().id),'wordpress')]"
```

Comment:
its is the same , but wpdbname is in azuredeploy.json and DbName is in mysql-replication.json . Do you want me use the same name in both files  ?